doc/examples/index.rst

```diff
@@ -0,0 +1,15 @@
+.. tutorials:
+
+Tutorials
+==========
+
+The following executable examples are available.
+
+Note that all of these tutorials are in the MyST format (`md`), and if downloaded can be opened as a Jupyter Notebook in Jupyter Lab ("Open with Notebook")
```

Comment:
Does that need `jupytext` to work?

Reply:
Ah, actually yes, you need to set up jupytext: https://jupytext.readthedocs.io/en/latest/paired-notebooks.html

I suppose if that's a problem I'd be fine w/ converting them back to notebooks,